doc/source/report/overview.rst

```diff
@@ -148,7 +148,38 @@ whole project can be divided into 6 sub-projects:
 -  **Strain:** A strain here is a genetic variant or sub-type of of the
    more general mouse population.
 
-5. References
+5. Data 
+-------
+The original data includes two folders, intervals and txy_coords, and a npy file named all_features_mousedays_11bins.
```

Comment:
I would use the term `data` instead of `original data`.  I also prefer the more standard term `directory` over `folder`.  (For more background, you might be interested in:  https://blogs.msdn.microsoft.com/oldnewthing/20110216-00/?p=11473/)

Since you are talking about the directory layout, you may want to indicate where these directories can be found.